tests/python/gaia-ui-tests/gaiatest/form_controls/binarycontrol.py

```diff
@@ -0,0 +1,58 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from abc import ABCMeta, abstractproperty
+from marionette_driver import Wait, expected
+from gaiatest.form_controls.form_control import FormControl
+
+
+class BinaryControl(FormControl):
+    # This represent elements like Switches and Checkboxes
+
+    __metaclass__ = ABCMeta
```

Comment:
This is needed to use the `@abstractproperty` . This is [documented standard python](https://docs.python.org/2.7/library/abc.html?highlight=abstractmethod#abc.abstractmethod)